deps.edn

```diff
@@ -62,5 +59,14 @@
            :format {:extra-deps {cljfmt/cljfmt {:mvn/version "0.8.0"}}
                     :main-opts ["-m" "cljfmt.main" "check"]}
 
-           :ffix {:extra-deps {cljfmt/cljfmt {:mvn/version "0.8.0"}}
-                  :main-opts ["-m" "cljfmt.main" "fix"]}}}
+           :ffix {:extra-deps {cljfmt/cljfmt {:mvn/version "0.7.0"}}
+                  :main-opts ["-m" "cljfmt.main" "fix"]}
+
+           :build {:deps {io.github.clojure/tools.build {:git/tag "v0.6.6" :git/sha "4d41c26"}
```

Comment:
what would you propose as update strategy for these dependencies?

Reply:
I would propose always to use the latest...should be stable or what do you mean?